iocore/net/QUICNetVConnection.cc

```diff
@@ -2412,6 +2418,26 @@ QUICNetVConnection::_handle_periodic_ack_event()
   }
 }
 
+SSL *
+QUICNetVConnection::_get_ssl_object() const
+{
+  auto tls = dynamic_cast<QUICTLS *>(this->_hs_protocol);
```

Comment:
Can this be anything other than `QUICTLS`?

Reply:
No, it's always `QUICTLS` at the moment because QUIC version 1 depends on TLS 1.3. QUIC version 2 may use something else.

I was thinking it's better to use `dynamic_cast` because version 2 may not use TLS, but on second thought, we should have another netvc type if version 2 doesn't use TLS. Because having TLSBasicSupport doesn't make sense if it doesn't use TLS. It would be tricky if version 2 supported multiple handshake protocols, but that wouldn't happen soon.

I'm going to change it to static_cast.